data/archiver/listener.py

```diff
@@ -43,10 +43,15 @@ def _data_archiver_message_handler(self, body: dict, msg: Message):
 
             result = Archiver(ingest_cli, AwsS3()).start(req)            
             self.logger.info(f'Archived data for submission uuid {req.sub_uuid}')
-            
+
+        except ValueError as e:
+            error_msg = f'Invalid data archiving request {body}: {str(e)}'
+            self.logger.debug(error_msg)
+            result = DataArchiverResult(req.sub_uuid, success=False, error=error_msg)
```

Comment:
When would we get this `ValueError` exception? Are you sure that the `req` variable is not contains a `None` value?

Reply:
last push should fix issue.